src/com/yetanalytics/lrs_admin_ui/handlers.cljs

```diff
@@ -211,7 +214,7 @@
    ;;extract the error and present it in a notification. If 401 or 0, log out.
    (let [message (if (= status 0)
                    "Could not connect to LRS!"
-                   (or (:error response)
+                   (or (response "error")
```

Comment:
huh?

Reply:
verifying this...

Reply:
I think it got through because there's a fallback for when it's not present.  And the  "fixed" behavior may be undesirable because, well, do we really want that monstrosity showing up as a toast notification?

Reply:
no, we may want some kind of exception wrapper like "Bad Input" or something

Reply:
I made it show the full error if <100 characters, else the former behavior ("An unexpected error has occurred!")